backend/prompt_studio/prompt_studio_registry/prompt_studio_registry_helper.py

```diff
@@ -71,6 +71,12 @@ def frame_spec(tool: CustomTool) -> Spec:
                 "default": False,
                 "description": "Enables single pass extraction",
             },
+            "enable_highlight": {
+                "type": "boolean",
+                "title": "Enable highlight",
+                "default": False,
+                "description": "Enables highlight",
```

Comment:
NIT: @Deepak-Kesavan elaborate on the description - how will this affect a non MRQ user?

Reply:
@chandrasekharan-zipstack Let it be like this for now. I am planning to enhance this after checking with @vishnuszipstack and remove this setting itself. We can internally handle this based on if MRQ is selected.